programming-fundamentals/chapter10.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Chapter 10: Exploring Further"
+date: "2023-11-08"
+author: "Andrei Guevorkian"
+illustrator: "Dengyijia Liu"
+questions:
+  - q1-tag: |
+      q1-text
+  - q2-tag: |
+      q2-text
+  - q3-tag: |
+        q3-text
+---
+
+
+## Table of Contents
```

Comment:
@hpyhacking I need your opinion on the table of contents. Do you think it's a good idea to list out all these fields and give a brief introduction to them?